app/views/welcome/_feed.html.erb

```diff
@@ -1,13 +1,17 @@
+<%= javascript_tag do %>
+  var unread_count = <%= current_user.unread_notifications_count %>
+<% end %>
 <div class="row">
   <div class="col-md-6">
     <h1>Новости</h1>
+    <% unread_text = "Непрочитанные" + if (current_user.unread_notifications_count > 0) then " (#{current_user.unread_notifications_count})" else "" end %>
```

Comment:
`current_user.unread_notifications_count` can be extracted to a variable. It'll be cleaner and if there's a lot of such code it should be somewhat faster.